app/javascript/src/components/Authentication/Login.jsx

```diff
@@ -39,43 +38,37 @@ const Login = ({ history }) => {
         <h2 className="mb-5 text-3xl font-extrabold text-center text-gray-800">
           Sign In
         </h2>
-        <form
-          className="w-full p-8 space-y-6 bg-white border rounded-md shadow"
+        <Formik
+          initialValues={{
+            email: "",
+            password: "",
+          }}
           onSubmit={handleSubmit}
```

Comment:
Can we have client-side validation for login and signup forms. 

- You can look more into [formik validation](https://formik.org/docs/guides/validation).
- Validation only needs to start once the user clicks the submit button and then shown always until the error is fixed. You can refer to PR https://github.com/bigbinary/neeto-desk-web/pull/10201 to know more.
- You can use validation schema using yup. ([Reference](https://formik.org/docs/tutorial#schema-validation-with-yup))

Reply:
Done.